biosimulations/apps/dispatch/src/app/components/help/about/about.component.html

```diff
@@ -256,8 +256,8 @@
     </biosimulations-text-page-content-section>
```

Comment:
Should we distinguish the teams for each app separately? I thought that we were going to use Biosimulations team for all the apps?

Reply:
Let's distinguish separate teams for BioSimulations and BioSimulators. Right now, they are the same people. That won't necessarily be the case. More importantly, I think its clearer for users to say that BioSimulations was developed by the BioSimulations team and BioSimulators was developed by the BioSimulators team.

Reply:
That makes sense definitely.

Reply:
Should we also do the same for the runBioSimulations? That at least seems to be more of a subset of BioSimulations